docs/Microservices/RPS/securityRPS.md

```diff
@@ -1,44 +1,49 @@
+In a default docker or Kubernetes deployment, RPS will save the Remote Admin Password to the deployed Vault instance.  If Vault is not configured, RPS will not use profiles that specify a randomly generated password.  This is to prevent the loss of access to a configured Intel&reg; AMT device.
 
-### 2 Provisioning Certificate
-This certificate is unique per owned domain that has Intel&reg; AMT devices located and RPS needs to provision.  This certificate must be derived from a root certificate whose hash matches one of the trusted provisioning root certificate hashes that is listed in the Intel&reg; AMT device firmware.  Generally, the provisioning certificate is purchased from a trusted certificate authority (VeriSign, GoDaddy, Comodo, etc).  The full list of supported CAs based on Intel&reg; AMT version are listed [here](https://software.intel.com/sites/manageability/AMT_Implementation_and_Reference_Guide/WordDocuments/rootcertificatehashes.htm).  This certificate must contain the leaf certificate, root certificate, and all of the intermediate certificates to form a complete certificate chain.  Additionally, the certificate file must also include the private key for the certificate (.pfx format).  The leaf certificate for the provisioning certificate must match the domain suffix that the Intel&reg; AMT device is connected as specified by DHCP option 15 or the Truted DNS Suffix in the Management BIOS Extentions (MEBx).  Matching this is one of the ways in which the Intel&reg; AMT firmware establishes trust with RPS.  
-RPS reads the location of the Provisioning Certificate from the configuration file and will then fetch the certificate when it is needed to activate an Intel&reg; AMT device.  RPS doesn't ever save or export the Provisioning Certificate anywhere.
+### 2 MEBx Password
+The Management Engine BIOS Extension (MEBx) password is the password that protects the pre-boot menu option that provides access to Intel&reg; AMT settings.  To use this password a user needs to have physical access to the device.  It is highly recommended to change this password from the factory default settings upon receiving a new Intel&reg; AMT device.  A RPS profile provides an option for either specifying a static password that is used for all devices configured with a given profile or a randomly generated password can be assigned uniquely per device.  The MEBx password set in each device is saved in Vault.
+While a randomly generated password is more secure, in this case there is risk that if the Vault database is lost, access to the Intel&reg; AMT device could be very difficult to recover.  It is recommended to use the high availability and backup options provided by the Vault solution to ensure that these secrets are not lost.
```

Comment:
Do we want to mention what is involved with recovery? Truck roll, clear cmos, etc...